packages/forklift-console-plugin/src/modules/Providers/utils/validators/common.ts

```diff
@@ -74,7 +77,7 @@ export function validateFingerprint(fingerprint: string) {
 }
 
 export function validateK8sName(k8sName: string) {
-  return DNS_SUBDOMAINS_NAME_REGEXP.test(k8sName);
+  return DNS_SUBDOMAINS_NAME_REGEXP.test(k8sName) && k8sName?.length <= 253;
```

Comment:
```suggestion
  return DNS_SUBDOMAINS_NAME_REGEXP.test(k8sName);
```

the change in https://github.com/kubev2v/forklift-console-plugin/pull/713/files#r1314450788 make it unnecessary to check the length here

Reply:
> The current code that I used was copied from backend forklift validation

I can't find the reference code in forklift, can you add a link to the reference implementation in forklift ?